webapp/src/Form/Type/ContestType.php

```diff
@@ -28,9 +28,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder->add('name', TextType::class);
         $builder->add('activatetimeString', TextType::class, [
             'label' => 'Activate time',
+            'help' => 'Time when the contest becomes visible for teams to join. Must be in the past to enable judging of jury submissions.',
```

Comment:
Is the second sentence true? Or should it be "to enable submission of jury submissions"? Honest question, do not know.

Reply:
I don't know either. Figured making a potentially wrong statement is the best way to find out :P

Reply:
Indeed, you can't submit as jury before activate time. But if there are submissions (since you moved activatetime after submitting), we'll keep judging.